ci/generate-workflows.py

```diff
@@ -177,6 +177,7 @@ def __init__(
         continuous_only=False,
         app_image=False,
         is_tag=False,
+        priorize=False,
```

Comment:
spelling here. This is actually quite awkward to spell/type, so how about we rename it to `hi_pri` instead? :-)

```suggestion
        hi_pri=False,
```

Reply:
Sure